components/deferComponentRender.tsx

```diff
@@ -19,6 +19,7 @@ type DeferredRenderWrapperState = {
  */
 export default function deferComponentRender<ComponentProps>(WrappedComponent: React.ComponentType<ComponentProps>, PreRenderComponent: React.ReactNode = null) {
     class DeferredRenderWrapper extends React.PureComponent<ComponentProps, DeferredRenderWrapperState> {
+        mounted = false;
```

Comment:
can we use ref here. 0/5

Reply:
@M-ZubairAhmed is there any benefit of using a ref here?

Reply:
there is no benefit of using ref vs var here. Using ref is just a pattern, i guess??

Reply:
@M-ZubairAhmed in my mind, using a ref makes sense in a functional component to mimic class variables, I mean you have no other way, but in a class I don't see why use it like that.